src/relay/op/contrib/ethosu/common.cc

```diff
@@ -75,6 +78,56 @@ Array<IndexExpr> EthosuInferKernelOutput(Array<IndexExpr> ifm_shape, String ifm_
   return output_shape;
 }
 
+DataType DataTypeFromString(const String& dtype) {
+  DLDataType dl_dtype = tvm::runtime::String2DLDataType(dtype);
+  return DataType(dl_dtype);
+}
+
+void CheckDataType(const TypeReporter& reporter, const DataType& data_type,
+                   const std::unordered_set<DataType>& allowed_data_types,
```

Comment:
I would have used `std::initializer_list` instead of `std::unordered_set`. It should be more efficient in this case,
it doesn't require allocation in the free store, and the number of elements is small to benefit the fast retrieval of `std::unordered_set`.

Reply:
Good point, I've used an initializer list instead, thanks!